src/main/resources/init_db.sql

```diff
@@ -1,9 +1,16 @@
-CREATE SCHEMA IF NOT EXISTS `taxi_service` DEFAULT CHARACTER SET utf8;
-USE `taxi_service`;
+CREATE SCHEMA IF NOT EXISTS taxi_service_database DEFAULT CHARACTER SET utf8;
+USE taxi_service_database;
 
-CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+CREATE TABLE taxi_service_database.`manufacturer`(
+            `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+            `name` VARCHAR(255) NULL,
+            `country` VARCHAR(255) NULL,
+            `is_deleted` TINYINT NOT NULL DEFAULT 0,
+            PRIMARY KEY (`id`));
+
+CREATE TABLE taxi_service_database.`driver` (
```

Comment:
```suggestion
CREATE TABLE taxi_service_database.`drivers` (
```
table names should be plural according to convention